service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -208,4 +209,19 @@ public Map<String, Object> removeCaseNote(CaseData caseData, String migrationId,
         return Map.of("caseNotes", caseNoteService.removeCaseNote(caseNoteIdToRemove, caseData.getCaseNotes()));
     }
 
+    public void verifyGatekeepingOrderUrgentHearingOrder(CaseData caseData, String migrationId, String fileName) {
+        if (caseData.getUrgentHearingOrder() == null) {
+            throw new AssertionError(format(
+                "Migration {id = %s, case reference = %s}, GateKeeping order - Urgent hearing order not found",
+                migrationId, caseData.getId()));
+        }
+
+        if (isNotEmpty(fileName)) {
```

Comment:
should we skip the checking if fileName is not supplied?

Reply:
hmmm.... just want some flexibility here. If we don't need to check it next time, we can simply ignore this field.

Reply:
maybe make a new method and write comments for the method usage? 
`public void verifyGatekeepingOrderUrgentHearingOrder(CaseData caseData, String migrationId)`

Reply:
done